pyccel/codegen/printing/cwrappercode.py

```diff
@@ -245,11 +244,70 @@ def _print_PyModule_AddObject(self, expr):
             var = f'(PyObject*) {var}'
         return f'PyModule_AddObject({expr.mod_name}, {name}, {var})'
 
+    def _print_PyCapsule_New(self, expr):
+        name = expr.capsule_name
+        var  = self._print(ObjectAddress(expr.API_var))
+        return f'PyCapsule_New((void *){var}, "{name}", NULL)'
+
+    def _print_PyCapsule_Import(self, expr):
+        name = expr.capsule_name
+        return f'(void**)PyCapsule_Import("{name}", 0)'
+
+    def _print_PyModule_Create(self, expr):
+        return f'PyModule_Create(&{expr.module_def_name})'
+
+    def _print_ModuleHeader(self, expr):
+        mod = expr.module
+        name = mod.name
+
+        # Print imports last to be sure that all additional_imports have been collected
+        imports  = [*module_imports, *mod.imports]
+        for i in imports:
+            self.add_import(i)
+        imports  = ''.join(self._print(i) for i in imports)
+
+        function_signatures = ''.join(self.function_signature(f, print_arg_names = False) + ';\n' for f in mod.external_funcs)
```

Comment:
If I am not mistaken one-line statements in the printer should end with a newline character, preceded by a semicolon in C. Do we not follow the same convention in this module?

Reply:
The convention is respected for one-line statements, however `self.function_signature` is used in 2 places:
1. Declaration (one-line statement) `void my_func();`
2. Definition (not one-line statement) `void my_func() { ... }`

So in this case the addition of `;\n` is left to the calling function

Reply:
`self.function_signature` mostly relies on the definition coming from ccode